MsCorePkg/Library/SecureBootKeyStoreLib/SecureBootKeyStoreLib.c

```diff
@@ -0,0 +1,99 @@
+/** @file PlatformKeyLib.c
+
+  Copyright (C) Microsoft Corporation. All rights reserved.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Uefi.h>
+#include <UefiSecureBoot.h>
+#include <Guid/ImageAuthentication.h>
+#include <Library/SecureBootVariableLib.h>
+
+#include <Library/DebugLib.h>
+#include <Library/MemoryAllocationLib.h>

```

Comment:
Can you check if `DebugLib` and `MemoryAllocationLib` are actually needed?

Reply:
Confirmed unnecessary. Next commit will remove. Thanks!